### ISSUE TITLE
[Godot 4.0 beta 17 & RC1] Microphone input does not work

### ISSUE THREAD

**Issue by OverloadedOrama** [godotengine/godot#72892]
### Godot version

4.0 beta 17, 4.0 rc1

### System information

openSUSE Tumbleweed (PulseAudio) and Windows 10 (WASAPI), compatibility renderer

### Issue description

Microphone input does not work at all in beta 17 and rc1, while it was working fine in beta 16. `Enable input` is toggled on in the project settings. Another interesting thing I noticed is that, on beta 16, when I have an AudioStreamPlayer node in the scene and I enable `Playing`, I can hear my microphone input normally on the Godot editor. On beta 17 and  rc1 however, when I click to enable `Playing`, it immediately becomes disabled.

These tests were being done on my Linux openSUSE Tumbleweed computer, but I also had this issue with beta 17 on Windows 10.

### Steps to reproduce

Simply go to the Audio tab on the bottom and add a new bus with the `Recording` effect. Then, create an empty scene and add an AudioStreamPlayer node, and add the new bus we just created. Also click `Autoplay` so that the stream will be heard when we run the project. Go to the project settings and enable `Enable input` and restart the editor. Run the project and talk to the microphone. On beta 16, you will hear your microphone input. On beta 17 and rc1, nothing can be heard.

### Minimal reproduction project

[MicrophoneIssue.zip](https://github.com/godotengine/godot/files/10686569/MicrophoneIssue.zip)

**Comment by Calinou** [comment]
You may need to change your code accordingly to https://github.com/godotengine/godot/pull/69120.

**Comment by OverloadedOrama** [comment]
The MRP I uploaded has no code at all, it's only a scene with a single AudioStreamPlayer inside.

**Comment by akien-mga** [comment]
Confirmed on Mageia 9.

With 4.0 beta 16, it errors:
```
ERROR: PulseAudio: pa_stream_disconnect error: Bad state
   at: capture_finish_device (drivers/pulseaudio/audio_driver_pulseaudio.cpp:741)
```
but the microphone input works.

With 4.0 beta 17, no error, but no microphone input.

**Comment by akien-mga** [comment]
I confirmed that it's a regression from #69120, reverting it fixes the issue. I don't see right away what would cause the issue but there must be a typo, or some call that still happens through an outdated StringName.
CC @souplamp

**Comment by OverloadedOrama** [comment]
I think I managed to find the issue. #69120 renamed `capture_start()` and `capture_stop()` to `input_start()` and `input_stop()` respectively, but only in certain scripts, therefore leaving the old names to co-exist with the new ones. I replaced every single instance of `capture_start()` and `capture_stop()` to their new names and microphone input seems to be working fine now. I will open a pull request shortly, but I'm not sure if there are more renames that still need to be done.
